### PR TITLE
Add description and maxLengths for headers

### PR DESCRIPTION
**Pull request by rebekaburnett**

### CODE REVIEW

**Comment by rebekaburnett** [comment]
This includes changes from https://github.com/vippsas/vipps-epayment-api/pull/95

File: docs/epayment.v1.yml [hunk 1]
```diff
@@ -817,42 +822,65 @@ components:
         minLength: 4
         maxLength: 6
         example: '123456'
-      description: "The Sales unit's MSN: The unique identifier for the sales unit."
+      description: |-
+        The merchant serial number (MSN) for the sales unit. Partners must always send
+        the Merchant-Serial-Number header, and we recommend that everyone sends it,
```
Comment (OFollan) [inline docs/epayment.v1.yml:827]: >  Partners must always send the Merchant-Serial-Number header, and we recommend that everyone sends it,

Actually, for ePayments API the MSN header is mandatory for all requests 😄

**Review by OFollan — CHANGES_REQUESTED** [review]

File: docs/epayment.v1.yml (hunk 1 above)
Comment (rebekaburnett) [reply docs/epayment.v1.yml:827]: Good to know!

**Review by OFollan — APPROVED** [review]
🚀

**Pull request merged**